lib/api/controllers/funnelController.js

```diff
@@ -709,4 +758,22 @@ function doAction(controller, request) {
   return ret;
 }
 
+/**
+ * Very straightforward function to check only if the version satisfies
+ * semver range major version
+ *
+ * @param {String} version
+ * @param {String} range
+ *
+ * @returns {Boolean}
+ */
+function satisfiesMajor (version, range) {
```

Comment:
Semver is very slow and we didn't need a complete range check. (this check has to be performed for every request)
```
Semver x 186,166 ops/sec ±4.04% (90 runs sampled)
Manual: only major x 683,741,550 ops/sec ±0.52% (82 runs sampled)
```